test/webhost/TestDocs.py

```diff
@@ -1,12 +1,16 @@
 import unittest
+import Utils
+import os
+
+import WebHost
 from worlds.AutoWorld import AutoWorldRegister
 
 
-class TestTutorial(unittest.TestCase):
+class TestDocs(unittest.TestCase):
+    tutorials_data = WebHost.create_ordered_tutorials_file()
```

Comment:
this should go into https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp